book/src/out-of-order-blocks.md

```diff
@@ -88,3 +88,27 @@ The validators already transmit their votes through gossip, so the golden ticket
 block can pick up the validator votes and any additional transactions that have
 not been encoded into the ledger.  These transactions could come from rejected
 forks.
+
+## Only Random Leaders
+
+If the random source is considered secure, it is possible to generate a leader
+schedule completely randomly.
+
+1. Leader 2 finds the golden ticket in slot 1
+2. Leader 2 proposes block for slot 2
+3. Block contains up to two PoH paths:
```

Comment:
I don't understand this one.  Do you propose that the PoH branches at the golden ticket hash?

Reply:
@rob-solana the ticket has a PoH path to a block, but the proposed block has a PoH path to another block.  The only point which those two are different is if Leader 2 attaches the block to the same block that was proposed concurrently with the golden ticket.

Reply:
I think I could use a picture...

Reply:
leader 2 runs PoH over the same slot that leader 1 is proposing a block.  Leader 2's block needs to pick a block to attach to.

Reply:
the PoH path for the golden ticket, starts from block 0 but doesn't connect to block 2, block 2 attaches directly to block 1.

Reply:
@rob-solana fixed i think, let me know what you think of the new explanation